api/DragEvent.json

```diff
@@ -47,6 +47,55 @@
           "deprecated": false
         }
       },
+      "DragEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/DragEvent/DragEvent",
+          "description": "<code>DragEvent</code> constructor",
+          "support": {
+            "webview_android": {
+              "version_added": false
+            },
+            "chrome": {
+              "version_added": "46"
+            },
+            "chrome_android": {
+              "version_added": false
+            },
+            "edge": {
+              "version_added": null
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox": {
+              "version_added": "3.5"
+            },
+            "firefox_android": {
+              "version_added": true
+            },
+            "ie": {
+              "version_added": null
```

Comment:
Is there a reason to believe the MDN data table is incorrect? It says IE support started with IE10. https://developer.mozilla.org/en-US/docs/Web/API/DragEvent/DragEvent#Browser_compatibility

Reply:
Oops, good catch! Fixed this in 6cf0f6a.